packages/volto/.eslintrc

```diff
@@ -37,6 +37,13 @@
     "react-hooks/exhaustive-deps": "warn",
     "react/react-in-jsx-scope": "off",
     "jsx-a11y/label-has-associated-control": "off",
+    "no-restricted-syntax": [
```

Comment:
@plone/volto-team we have to be more careful in the future with this things.

This is a breaking change, and should not be backported.  I totally overlooked it. I will take care and re-release.

@wesleybl We will need a breaking change notice written for 19, and modify the changelog everywhere (including GH release).

Reply:
https://github.com/plone/volto/pull/7401

Reply:
@wesleybl I just saw your PR! It's perfect, thanks!